### ISSUE TITLE
corefonts is broken for prefixes with special characters

### ISSUE THREAD

**Issue by DrDoctor13** [Winetricks/winetricks#995]
```
/home/bucks> WINEPREFIX=/home/bucks/Linux\ Games/Lutris/the-sims-2 winetricks corefonts
Using winetricks 20180513-next - sha256sum: c5438329b437e02416de6278346b5d0a8cb3d586415d069288fc85a300c9a02e with wine-3.7 (Staging) and WINEARCH=win32
Executing w_do_call corefonts
Executing load_corefonts 
Executing cabextract -q -d /home/bucks/Linux Games/Lutris/the-sims-2/dosdevices/c:/windows/temp/_corefonts /home/bucks/.cache/winetricks/corefonts/arial32.exe
Executing cp -f /home/bucks/Linux /home/bucks/Linux Games/Lutris/the-sims-2/dosdevices/c:/windows/Fonts/linux
cp: cannot stat '/home/bucks/Linux': No such file or directory
------------------------------------------------------
Note: command cp -f /home/bucks/Linux /home/bucks/Linux Games/Lutris/the-sims-2/dosdevices/c:/windows/Fonts/linux returned status 1. Aborting.
------------------------------------------------------

```

**Comment by austin987** [comment]
The issue appears to be the finds in w_try_cp_font_files(). Solving this in non-bash is going to be a bit tricky, since we need to do a couple things for each file (remove similarly named files and copy the font in).

http://mywiki.wooledge.org/BashPitfalls#for_i_in_.24.28ls_.2A.mp3.29
https://stackoverflow.com/questions/22762816/how-to-iterate-null-separated-results-in-non-bash-shell

have some helpful info on possible workarounds.

**Comment by austin987** [comment]
Fixed by 7060a53
and now tests should prevent it via 62426e8
